multiapps-mta/src/main/java/org/cloudfoundry/multiapps/mta/resolvers/PropertiesResolver.java

```diff
@@ -171,8 +181,7 @@ protected Object resolveReferenceInDepth(Reference reference, Map<String, Object
                 currentProperty = resolveKeyInIterable(keyPart, (Collection<?>) currentProperty, deepReferenceKey);
                 keyPart = "";
             } else if (currentProperty instanceof Map) {
-                @SuppressWarnings("unchecked")
-                Object subProperty = MapUtils.getObject((Map<Object, ?>) currentProperty, keyPart);
+                @SuppressWarnings("unchecked") Object subProperty = MapUtils.getObject((Map<Object, ?>) currentProperty, keyPart);
```

Comment:
In the rest of the code @SuppressWarnings("unchecked") is on a separate row, please align

Reply:
+1